### PR TITLE
[BEAM-3257] Publish Python precommit test results to Jenkins, remove Maven based Python precommit.

### PR DESCRIPTION
**Pull request by lukecwik**
Publish python test results to Jenkins.

Delete Maven based Python precommit.

Note that this change supersedes #4689 

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand:
   - [ ] What the pull request does
   - [ ] Why it does it
   - [ ] How it does it
   - [ ] Why this approach
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

**Comment by lukecwik** [comment]
CC: @udim @robertwb

File: sdks/python/tox.ini [hunk 1]
```diff
@@ -50,7 +50,7 @@ commands =
   pip --version
   {toxinidir}/run_tox_cleanup.sh
   python apache_beam/examples/complete/autocomplete_test.py
-  python setup.py test
+  python setup.py nosetests --with-xunit
```
Comment (udim) [inline sdks/python/tox.ini]: Will each run's `nosetests.xml` output file overwrite the others'?

Comment (lukecwik) [reply sdks/python/tox.ini]: I believe it will since they will overwrite each other. You would only get the test results from the last run.

**Comment by aaltay** [comment]
Superseeded by #5045

**Pull request closed without merging**